src/app/features/genes/components/gene-evidence-rna/gene-evidence-rna.component.ts

```diff
@@ -71,13 +77,20 @@ export class GeneEvidenceRnaComponent implements AfterViewInit {
     this.initConsistencyOfChange();
   }
 
-  ngAfterViewInit() {
-    const hash = window.location.hash.substr(1);
-    if (hash) {
-      const target = document.getElementById(hash);
-      if (target) {
-        // TODO determine if there are async calls altering the offset height
```

Comment:
There is an async call to fetch the differential expressions data, which is then used to render the box plot. We need to wait for the box plot to be initialized before scrolling, otherwise the offset will be incorrect.

Reply:
looks great!